templates/account.php

```diff
@@ -289,18 +289,14 @@ class="nav-tab"><?php echo esc_html( $addons_text ) ?></a>
                                     <?php if ( ! $is_whitelabeled ) : ?>
                                         <?php if ( ! $is_paying ) : ?>
                                             <li>
-                                                <form action="<?php echo $fs->_get_admin_page_url( 'account' ) ?>" method="POST">
-                                                    <input type="hidden" name="fs_action" value="delete_account">
-                                                    <?php wp_nonce_field( 'delete_account' ) ?>
-                                                    <a class="fs-delete-account" href="#" onclick="if (confirm('<?php
-                                                        if ( $is_active_subscription ) {
-                                                            echo esc_attr( sprintf( fs_text_inline( 'Deleting the account will automatically deactivate your %s plan license so you can use it on other sites. If you want to terminate the recurring payments as well, click the "Cancel" button, and first "Downgrade" your account. Are you sure you would like to continue with the deletion?', 'delete-account-x-confirm', $slug ), $plan->title ) );
-                                                        } else {
-                                                            echo esc_attr( sprintf( fs_text_inline( 'Deletion is not temporary. Only delete if you no longer want to use this %s anymore. Are you sure you would like to continue with the deletion?', 'delete-account-confirm', $slug ), $fs->get_module_label( true ) ) );
-                                                        }
-                                                    ?>'))  this.parentNode.submit(); return false;"><i
-                                                            class="dashicons dashicons-no"></i> <?php fs_esc_html_echo_inline( 'Delete Account', 'delete-account', $slug ) ?></a>
-                                                </form>
+                                                <?php
+                                                    $view_params = array(
+                                                        'freemius'               => $fs,
+                                                        'is_active_subscription' => $is_active_subscription,

```

Comment:
@xiahengchen I started testing this out. Turns out the `$is_paying` and `$is_active_subscription` are mutually inclusive. When `$is_paying` is `false`, the `$is_active_subscription` will also be `false`. Our check works like that.

So when a license is active, i.e, `$is_active_subscription`, we don't actually show this block. Instead we show this.

<img width="364" alt="image" src="https://user-images.githubusercontent.com/1623381/166147056-f5f27fbf-4132-4446-ad55-bbce07ca60bf.png">

Click on the "Deactivate License" button actually tells us a similar message.

<img width="516" alt="image" src="https://user-images.githubusercontent.com/1623381/166147081-af2b2ab5-fcca-42d1-a9bd-fe6d25488473.png">

So, we don't need this extra check and this extra variable. It was an oversight from our end. Hence, kindly remove this variable from template params.

Instead, what we do want to introduce is, if there's a license available. Like check the state of account here. I can activate the "Ultra License" because it was deactivated, but not yet fully disconnected.

<img width="832" alt="image" src="https://user-images.githubusercontent.com/1623381/166147274-16cacf80-cc36-420e-bef8-54e536516634.png">

From the code, the check works like this:

```php
$available_license = $fs->is_free_plan() && ! fs_is_network_admin() ? $fs->_get_available_premium_license( $site->is_localhost() ) : false;
```

You will find this on line `545` of this file. So let us move the `$available_license` logic on the top and pass `is_license_available` to the template. If `$is_license_available` is present, then change the `$message`.